database/migrations/2019_07_19_075325_rebuild__d_b.php

```diff
@@ -23,11 +23,10 @@ public function up()
             $table->char('location', 5);
             $table->string('name');
             $table->text('description');
-            $table->string('county')->nullable();
-            $table->string('district')->nullable();
+            $table->char('county',10)->nullable();
+            $table->tinyInteger('avg_rating')->nullable();;
```

Comment:
syntax error ";"

Reply:
I revise the syntax , thx !
If there is no problem anymore , please help me merge pull request . : )